ports/gdal/vcpkg.json

```
@@ -40,6 +42,10 @@
       "dependencies": [
         "libmariadb"
       ]
+    },
+    "system-libraries": {
+      "$supports": "!windows",
```

```suggestion
      "supports": "!windows",
```
Should be `supports`?

---

If "supports" was a supported field inside "features"... It is just documentation at the moment.

---

cc @strega-nil-ms

---

@JackBoosY cf. https://github.com/microsoft/vcpkg/issues/17651

---

Fine, please run `x-add-version`.